scss/navbar.scss

```diff
@@ -0,0 +1,22 @@
+.nav {
+  align-items: center;
+  background-color: $white;
+  box-shadow: 5px 7px 5px 0px $grey;
+  display: flex;
+  flex-wrap: wrap;
+  height: $nav-height;
+  justify-content: space-between;
+  padding: 5px 20px;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 1;
+
+  .navbar-logo {
```

Comment:
It's not necessary nesting this class definition, please move it outside of here